src/seedu/addressbook/data/person/Person.java

```diff
@@ -16,6 +16,9 @@
     private Address address;
 
     private final UniqueTagList tags;
+    
+    private int sequenceNum;
```

Comment:
Can you explain how to use this sequence number? 
Hints: if its private how can it be accessed without a getter? Even if there is a getting if its not added to the interfaces how can it access via those class?